src/pages/FrontendMentor/Todo/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { LogoBox, MainBox, Rows } from './Todo.styled'
+import { TopRow } from './Todo.styled'
+import { MainContainer } from './Todo.styled'
+import { useState } from 'react'
+import ToDoCard from '../../../components/ToDo/ToDoCard'
+import ToDoList from '../../../components/ToDo/ToDoList'
+import ToDoForm from '../../../components/ToDo/ToDoForm'
+
+type todo = {
+  key: string
+  task: string
+  completed: boolean
+}
+
+const initialList: todo[] = [
+  {
+    key: '1',
+    task: 'Buy groceries',
+    completed: false,
+  },
+  {
+    key: '2',
+    task: 'Study Finals',
+    completed: false,
+  },
+]
+
+const Todo = () => {
+  const [todos, setTodos] = useState(initialList)
```

Comment:
add in state of todos in useState -> const [todos, setTodos] = useState<Todo[]>(initialList)